test/unit/ngsiv2/ngsiService/subscriptions-test.js

```diff
@@ -75,14 +75,13 @@ describe('Subscription tests', function() {
         nock.cleanAll();
 
         iotAgentLib.activate(iotAgentConfig, function() {
-            // FIXME: change once NGISv2 device provisioning is implemented
             contextBrokerMock = nock('http://192.168.1.1:1026')
                 .matchHeader('fiware-service', 'smartGondor')
                 .matchHeader('fiware-servicepath', '/gardens')
-                .post('/v1/updateContext',
-                    utils.readExampleFile('./test/unit/examples/contextRequests/createMinimumProvisionedDevice.json'))
-                .reply(200,
-                    utils.readExampleFile('./test/unit/examples/contextResponses/createProvisionedDeviceSuccess.json'));
+                .post('/v2/entities',
+                    utils.readExampleFile('./test/unit/ngsiv2/examples/' +
+                        'contextRequests/createMinimumProvisionedDevice.json'))
+                .reply(201);
```

Comment:
The right code is 204 (see http://telefonicaid.github.io/fiware-orion/api/v2/stable/)

It is not important (I guess that the IOTA agent looks for anything between 200 and 299) but better to use the right one in the tests ;)

(Sorry for this after-LGTM comment... the https://github.com/telefonicaid/iotagent-json/pull/284/files#r195453831 comment has made me think if PR in the library have the same situation...)

Reply:
Wrong comment. NTC